<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>830</width>
    <height>594</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TinyGIS</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>830</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>FisrtTask</string>
    </property>
    <addaction name="actionLoadE00File"/>
    <addaction name="actionGenerateRoute"/>
   </widget>
   <widget class="QMenu" name="menuDisplay">
    <property name="title">
     <string>SecondTask</string>
    </property>
    <addaction name="actionLoadDemandShp"/>
    <addaction name="actionLoadSupplyShp"/>
    <addaction name="actionCalculateAccessibility"/>
   </widget>
   <widget class="QMenu" name="menuThirdTask">
    <property name="title">
     <string>ThirdTask</string>
    </property>
    <addaction name="actionLoadPointShp"/>
    <addaction name="actionCalculateKde"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuDisplay"/>
   <addaction name="menuThirdTask"/>
  </widget>
  <action name="actionLoadE00File">
   <property name="text">
    <string>LoadE00File</string>
   </property>
  </action>
  <action name="actionLoadDemandShp">
   <property name="text">
    <string>LoadDemandShp</string>
   </property>
  </action>
  <action name="actionGenerateRoute">
   <property name="text">
    <string>GenerateRoute</string>
   </property>
  </action>
  <action name="actionLoadSupplyShp">
   <property name="text">
    <string>LoadSupplyShp</string>
   </property>
  </action>
  <action name="actionCalculateAccessibility">
   <property name="text">
    <string>CalculateAccessibility</string>
   </property>
  </action>
  <action name="actionLoadPointShp">
   <property name="text">
    <string>LoadPointShp</string>
   </property>
  </action>
  <action name="actionCalculateKde">
   <property name="text">
    <string>CalculateKde</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
